bowtie.wdl

```diff
@@ -0,0 +1,43 @@
+task Bowtie_se {
+    File reference
+    File read1
+    String sam
+    String? precommand
+    Int? threads
+    String? otherOptions
```

Comment:
I disagree with other options.
If other options are required, another pull request should be made. Or alternatively, the entire command should be written out (which will take a lot of time, so may not be the best option for now).